packages/frontend-shared/src/components/Card.vue

```diff
@@ -9,6 +9,12 @@
     data-cy="card"
     @click="emits('click')"
   >
+    <div
+      v-if="title === t('testingType.component.name')"
+      class="ribbon absolute top-0 right-0 text-teal-600"
```

Comment:
Let's put `aria-hidden` on this so that it isn't announced by screen readers at this point in the DOM - it is encountered after the first card and at the start of the second card, but there's nothing to tell the user which it refers to or what is in beta.

Probably, as part the card title we could add something like this at the end:

```html
<span class='visually-hidden'>Beta</span>
```

Which would seem to be the clearest way to reproduce the effect of the ribbon when listening to the page.

Reply:
Nice idea, will do.

We should do a screen test run through of the entire app.